mava/configs/logger/ff_isac.yaml

```diff
@@ -0,0 +1,9 @@
+defaults:
+    - base_logger
+
+system_name: ff_isac
+
+# --- Neptune options ---
+kwargs:
+  neptune_project: Instadeep/Mava
+  neptune_tag: [ff-isac, mabrax]
```

Comment:
Nit, can we remove all the Neptune overrides from here please. The logger tags etc should all be getting set in `base_logger.yaml` for now similarly to other systems.